android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt

```diff
@@ -317,7 +317,7 @@ fun HomeListChildView(
 
 				items(uiState.listingItems.listings.size) {
 					val listingSummary = uiState.listingItems.listings[it]
-					val listingImage = uiState.listingItems.listingsImages[it]
+					val listingImage = if (uiState.listingItems.listingsImages.size > it ) uiState.listingItems.listingsImages[it] else null
```

Comment:
Good catch, I think a cleaner more "kotlin" way of doing this is
`uiState.listingItems.listingsImages.getOrNull(it)`